pkg/update/update.go

```diff
@@ -193,13 +193,17 @@ func PartitionedRollingUpdateStrategy(perPodVerificationFunc func(*UpdateSts, in
 			stsName := sts.Name
 			stsNamespace := sts.Namespace
 
-			// If pod already updated, we are probably retrying a failed job
-			// attempt. Best not to redo the update in that case, especially the sleeps!!
-			if err := perPodVerificationFunc(updateSts, int(partition), l); err == nil {
-				l.Info("already updated, skipping sleep", "partition", partition)
-				skipSleep = true
-				continue
-			}
+			/*
+				// TODO why are we doing this?
```

Comment:
@chrisseto @alinadonisa I am trying to reuse the update code in order to do a rolling update of the cluster.  So two things are happening.  

1. This code block is stopping the update from happening
2. The sts is not updating so we are not rolling the pods.

So ideas
1. don't reuse the update code
2. update the sts so that it does an update
3. change the code somehow so we are inserting the capability to delete a pod

Thoughts?